.github/workflows/ci.yml

```diff
@@ -11,6 +11,10 @@ on:
       - master
       - "release*"
 
+concurrency:
```

Comment:
@PProfizi what do that mean?

Reply:
@cbellot000 
> When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be pending. Any previously pending job or workflow in the concurrency group will be canceled. To also cancel any currently running job or workflow in the same concurrency group, specify cancel-in-progress: true.

see [here](https://docs.github.com/en/actions/using-jobs/using-concurrency)
It means committing to your PR will cancel an already running workflow for the same PR.
When you work like me and make frequent commits, you often get the same workflow running multiple times in parallel, once for every commit, instead of just for the latest commit.